core-image.html

```diff
@@ -179,12 +212,94 @@
        * @type number
        * @default null
        */
-      height: null
+      height: Number
+    },
+
+    configure: function() {
+      return {
+        src: '',
```

Comment:
I used empty string to mimic the behavior of native `<img>`. Without an explicit `src` attribute provided, the image's `src` property appears to be an empty string. I'll update the documentation.